File: Bubblesort.cpp
#include<bits/stdc++.h>
using namespace std;

void printarray(int arr[], int n){
    for(int i = 0; i<n ; i++){
        cout<<arr[i]<<"  ";
    }
}

void sortarray(int arr[], int n){
for(int i = 0 ;i< n ;i++)
 
for(int j = i + 1 ;j<n ;j ++){
    if ( arr[i]>arr[j]){
        int temp = arr[i];
        arr[i] = arr[j];
        arr[j] = temp ;
    }
}
}


int main(){
    cout<<"Bubble sort\n";
    int arr[] = {6,9,0,2,4,1,3};
    int n = 7;
    cout<<"Unsorted Arra is:";
    printarray(arr, n);

    sortarray(arr,n);
    cout<<"\nSorted Array is :";
    printarray(arr,n);
}

File: BubbleSort.java

public class BubbleSort{ 


   static void printarray(int arr[], int n)
{
    int i ;
    for(i = 0 ;i<n ;i++) {
        System.out.print(arr[i]+ "\t");
    }
}

static void sortarray(int arr[], int n){
    int i ;
for(i = 0 ;i<n ;i++){
    int j,temp ;
    for(j = 0;j<n-1;j++){
        if(arr[j]>arr[j+1]){
           temp = arr[j];
           arr[j] = arr[j+1];
           arr[j+1] = temp ;
        }
    }
}
}



public static void main(String[] args) {
    System.out.println("Bubble sort. ");
    int[] arr = {5,3,7,9,6,4};
    int n = 6;
    System.out.print("Unsorted array is :");
    printarray(arr, n);
    sortarray(arr,n);
    System.out.print("sorted Array is: ");
    printarray(arr,n);
}
}